src/containers/Page.jsx

```diff
@@ -137,10 +140,10 @@ class Page extends Component {
     if (document && document.body) {
       document.body.classList.remove('noScroll');
```

Comment:
We are making the `body` not scrollable when you open the modal using javascript.

Reply:
You could use a ref and attach the scroll event when there the modal is not open only. We can come back to this later since we should use refs minimally.